GitUI/CommandsDialogs/SettingsDialog/Pages/AppearanceFontsSettingsPage.cs

```diff
@@ -39,6 +40,7 @@ protected override void PageToSettings()
             AppSettings.FixedWidthFont = _diffFont;
             AppSettings.Font = _applicationFont;
             AppSettings.CommitFont = _commitFont;
+            AvatarService.UpdateAvatarInitialFontsSettings();
```

Comment:
This should be done whenever the settings are updated (the settings file is watched) - definitely not here.

Reply:
Here is the only place where the setting used is updated so that was for me the best place to call the method making the code straightforward to understand.
Can you tell me the place you were thinking about because I don't see/find it (and in a sense, IMHO it confirms what I say in the first sentence 😉 )

Reply:
Run two instances of GE, change the setting in one of them, go to other one and find the settings page being inconsistent with the settings used.
I just know that there is a FileSystemWatcher triggering a settings reload. (That's why I created the `RunTimeSetting<T>` class and used it for view settings.)
I have not searched how this reload works in detail.
But rather move that call to FormBrowse, which updates lot of things after closing the settings dialog.